### DISCUSSION TITLE
Re-write test driver in Perl

### DISCUSSION THREAD

**Discussion by hwangswan** [khongsomeo/GPA-OOP#102] [Ideas]
Perl is considered faster than Bash. Why don't we give it a chance?
- Recode Perl test driver: silent mode, loading test input from a folder.
- Remove bash test if necessary

**Comment by trhgquan** [comment]
Nice. Remember that the test driver should only use [Perl's default modules](https://perldoc.perl.org/modules). Any other external modules could require an installation step, which we would **not** want to.
